controlplane/pkg/monitor/monitor_server.go

```diff
@@ -12,6 +14,11 @@ const (
 	INITIAL_STATE_TRANSFER = "INITIAL_STATE_TRANSFER"
 )
 
+const (
+	src_prefix = "SRC-"
+	dst_prefix = "DST-"
```

Comment:
Where these come from? We should maintain common constant definitions across all packages we generate. Please consider refactoring the other modules if needed.

Reply:
If these are related to the prefixes we happened to have selected in vppagent-dataplane... I'd prefer *not* to presume upon them, and they may at some point change...